hooks/configure

```diff
@@ -74,6 +74,34 @@ switch_service() {
 	esac
 }
 
+update_pi_config_value() {
+    path="$1"
+    key="$2"
+    value="$3"
+
+    if [ -z "$value" ]; then
+        # empty - unset an option
+        sed -i "/^[ \t#]*${key}=/ s/^#*/#/" "$path"
+    else
+        # non-empty - set an option
+        export key value
+        awk -v pat="^[ \t#]*$key=.*" --sandbox -f - "$path" >"${path}.tmp" <<'EOF'
```

Comment:
the awk looks fine on a first glance ...

@mvo5 indeed i know what sed -i does ... but due to the design of snapctl you always have to loop over all values so you do this 20 times in a row every time one of the values changes, i meant to rather do the mv after the complete loop ran (but perhaps i'm just over-cautious here) and run the loop itself on a tmp file.